src/main/kotlin/org/hyacinthbots/saffronstatus/database/migrations/v1.kt

```diff
@@ -6,9 +6,20 @@
 
 package org.hyacinthbots.saffronstatus.database.migrations
 
+import org.hyacinthbots.saffronstatus.database.entities.WatchedBotData
 import org.litote.kmongo.coroutine.CoroutineDatabase
+import org.litote.kmongo.exists
+import org.litote.kmongo.setValue
 
 @Suppress("UnusedPrivateMember")
 suspend fun v1(db: CoroutineDatabase) {
 	// TODO Fill with migration and set true in main function
+	with(db.getCollection<WatchedBotData>()) {
+		updateMany(
+			WatchedBotData::publishMessage exists false,
+			    setValue(
+			        WatchedBotData::publishMessage, false
+			)
```

Comment:
Some strange indentation afoot here. Try ctrl + alt + L or the below
```suggestion
			WatchedBotData::publishMessage exists false,
			setValue(WatchedBotData::publishMessage, false)
```